iai_semantic_maps/owl/kitchen.owl

```diff
@@ -2030,6 +2040,13 @@
         <knowrob:startTime rdf:resource="&iai-map;timepoint_0000000001"/>
         <knowrob:objectActedOn rdf:resource="&iai-map;iai_kitchen_kitchen_island_counter_top"/>
     </owl:NamedIndividual>
+    
+    <owl:NamedIndividual rdf:about="&knowrob;SemanticMapPerception_Island_MealTable">
```

Comment:
When @bbrieber will be adding relative poses for the semantic objects he will have to rename the stuff.
It's better to put a little bit of thought into this now than renaming stuff later.
Actually, I will let @bbrieber merge this PR and decide what matters, he has more experience.